module/mempool/entity_requests.go

```diff
@@ -0,0 +1,14 @@
+package mempool
+
+import "github.com/onflow/flow-go/engine"
+
+const DefaultEntityRequestCacheSize = 100_000
+
+// EntityRequestStore is a FIFO (first-in-first-out) size-bound queue for maintaining EntityRequests.
+// It is designed to be utilized at the common ProviderEngine to maintain and respond entity requests.
```

Comment:
```suggestion
// It is designed to be utilized at the common ProviderEngine to maintain and respond to entity requests.
```

Reply:
This implementation has been removed and consolidated with `HeroQueue` and `HeroStore`.